solr/licenses/netty-NOTICE.txt

```diff
@@ -162,9 +170,9 @@ This product optionally depends on 'JBoss Marshalling', an alternative Java
 serialization API, which can be obtained at:
 
   * LICENSE:
-    * license/LICENSE.jboss-marshalling.txt (GNU LGPL 2.1)
+    * license/LICENSE.jboss-marshalling.txt (Apache License 2.0)
```

Comment:
I checked this -- looks accurate.  Then I got curious and searched out entire licenses folder for "LGPL".  netty-transport-native-epoll-NOTICE.txt (and unix-common-NOTICE.txt) also makes a reference to LGPL but it's via the same jboss-marshalling line you are correcting here.  So I think they can also be updated.